src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/InterventionHelper.kt

```diff
@@ -0,0 +1,93 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity
+
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import java.util.UUID
+
+class InterventionHelper {
```

Comment:
Since all of the functions here are samples, what do you think about calling this class `InterventionFixtures` or `Fixtures`?

Reply:
Sure.  It looks like the Referral tests could make sure of this class so a more general name looks preferable. How about just "SampleData" ?